src/networkProviders/apiNetworkProvider.ts

```diff
@@ -146,17 +147,17 @@ export class ApiNetworkProvider implements INetworkProvider {
         return status;
     }
 
-    async sendTransaction(tx: ITransaction | ITransactionNext): Promise<string> {
+    async sendTransaction(tx: ITransaction): Promise<string> {
```

Comment:
I think `ITransactionNext` should have been deleted only when updating the `Transaction` class and use that here.

Reply:
I guess it's fine since you'll do it in the next PR.